lib/cisco_node_utils/cmd_ref/interface_portchannel.yaml

```diff
@@ -57,6 +57,15 @@ lacp_suspend_individual:
     auto_default: false
     default_value: true
 
+lacp_suspend_shut_needed:
+  # This is used to determine if the port-channel interface
+  # must be shutdown before setting lacp_suspend_individual
+  kind: boolean
+  N3k:
```

Comment:
Good suggestion.  I made the change to only shutdown the interface when needed on the `n9k` platform.

Changes Tested on: n3k, n9k, n5k, n7k